README.md

```diff
@@ -3,6 +3,55 @@
 A [Singer](https://singer.io) Tap to extract currency exchange rate
 data from [fixer.io](http://fixer.io).
 
+## How to use it
+
+`tap-fixerio` works to pull currency exchange rates from the Fixer API
+
+### Install and Run
+
+First, make sure Python 3 is installed on your system or follow these
+installation instructions for [Mac] or
+[Ubuntu].
```

Comment:
Line wrapping here is a little weird. I think `[Ubuntu].` should be on the previous line?